prov/tcp/src/tcpx_progress.c

```diff
@@ -59,19 +59,6 @@ static void tcpx_cq_report_xfer_fail(struct tcpx_ep *tcpx_ep, int err)
 	}
 }
 
-static void tcpx_report_error(struct tcpx_ep *tcpx_ep, int err)
-{
-	struct fi_eq_err_entry err_entry = {0};
-
-	tcpx_cq_report_xfer_fail(tcpx_ep, err);
```

Comment:
The change here means that we're no longer writing entries to the CQ for pending transfers.  Maybe that's been done through another path, but I think we'll want to audit the code for this.